src/daft-sql/src/modules/image/resize.rs

```diff
@@ -0,0 +1,86 @@
+use daft_dsl::{Expr, ExprRef, LiteralValue};
+use sqlparser::ast::{FunctionArg, FunctionArgOperator};
+
+use crate::{
+    error::SQLPlannerResult,
+    functions::{FromSQLArgs, SQLFunction},
+    unsupported_sql_err,
+};
+use daft_functions::image::resize::{resize, ImageResize};
+
+pub struct SQLImageResize;
+
+impl FromSQLArgs for ImageResize {
+    fn from_sql(
+        inputs: &[sqlparser::ast::FunctionArg],
+        planner: &crate::planner::SQLPlanner,
+    ) -> SQLPlannerResult<Self> {
+        let mut width = None;
+        let mut height = None;
+
+        for arg in inputs {
+            match arg {
+                FunctionArg::Named {
+                    name,
+                    arg,
+                    operator: FunctionArgOperator::Assignment,
+                } => match name.value.as_ref() {
+                    "w" | "width" => {
```

Comment:
Hmm how would be document this behavior? I'm guessing we're going to need a separate documentation page for SQL (or at least some kind of unified documentation across SQL and Python expressions) in which we should also document the argument naming behavior?